lib/utils/utils.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// MakeRequest makes an API request.
+// It returns a response object and an error object.
+func MakeRequest(payload interface{}, URL, method string) (*http.Response, error) {
+	// Build the request
+	b, err := json.Marshal(payload)
+	if err != nil {
+		return nil, fmt.Errorf("cannot marshal payload. error: %v", err)
+	}
+
+	body := bytes.NewReader(b)
+	req, err := http.NewRequest(method, URL, body)
+	if err != nil {
+		return nil, fmt.Errorf("cannot make HTTP request. error: %v", err)
```

Comment:
`cannot create HTTP request` seems to be a better wording for the error message.
`make` can be jumbled with performing a request.